specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-06-01/managedClusters.json

```diff
@@ -2222,6 +2222,10 @@
           "type": "boolean",
           "description": "Whether to enable managed AAD."
         },
+        "enableAzureRBAC": {
```

Comment:
Add example for this new property?

Reply:
I have created a new file for the example .\examples\ManagedClustersCreate_UpdateWithEnableAzureRBAC.json